extension/intents/tabs/tabs.js

```diff
@@ -556,3 +556,20 @@ if (!buildSettings.android) {
     },
   });
 }
+
+intentRunner.registerIntent({
+  name: "tabs.findOnPage",
+  async run(context) {
+    await browser.find.find(context.slots.query).then(found);
```

Comment:
You can use `const results = await browser.find.find(context.slots.query);`

Additionally I notice that if you find something that is scrolled off the page, then it will not actually scroll to that item. E.g., if I go to [this page](https://blog.rust-lang.org/2020/03/12/Rust-1.42.html) and say "find cargo on this page" it will highlight the term "cargo" but off-screen.

It looks like [browser.find.highlight](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/find/highlightResults) has an option to scroll. That could be called just after `browser.find.find`.

Reply:
Hi @ianb!
I made the changes but I couldn't make the scroll to work. I used the [`{ noScroll: false }`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/find/highlightResults) option . 
Is this the method you are referring to?

Reply:
Yes, when I experimented I found `browser.find.highlight({noScroll: false, rangeIndex: 0})` worked (I guess it has to know which of the selections to scroll to)

Reply:
That's it! Thanks :)